phpBB/styles/prosilver/theme/mentions.css

```diff
@@ -0,0 +1,92 @@
+/* -------------------------------------------------------------- /*
+    $Mentions
+/* -------------------------------------------------------------- */
+
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable selector-no-qualifying-type */
+
+/* Mention block
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+/* Mention dropdown
+---------------------------------------- */
+.mention-container { /* mention-container */
+	text-align: left;
+	border-radius: 2px;
+	position: absolute;
+	z-index: 999;
+	overflow: auto; /* placed here for list to scroll with arrow key press */
+	max-height: 200px;
+	transition: all 0.2s ease;
+}
+
+.mention-container ul { /* mention-list */
```

Comment:
again I think this should be mention-list class. may need to tweak the js for it though.